cmd/goal/node.go

```diff
@@ -92,6 +96,7 @@ func init() {
 	createCmd.Flags().BoolVarP(&newNodeIndexer, "indexer", "i", localDefaults.IsIndexerActive, "Configure the new node to enable the indexer feature (implies --archival)")
 	createCmd.Flags().StringVar(&newNodeRelay, "relay", localDefaults.NetAddress, "Configure as a relay with specified listening address (NetAddress)")
 	createCmd.Flags().StringVar(&listenIP, "api", "", "REST API Endpoint")
+	createCmd.Flags().BoolVar(&newNodeFullConfig, "full-config", false, "Store full config file")
```

Comment:
Should this default to true?

Reply:
I preserve the old behavior by default, mb some folks like it